docker-compose.yml

```diff
@@ -10,15 +10,16 @@ volumes:
 
 services:
   mysql:
-    image: mysql:5
+    image: mysql:8
```

Comment:
MySQL 8 introduces a new default authentification method (see https://dev.mysql.com/blog-archive/upgrading-to-mysql-8-0-default-authentication-plugin-considerations/).
To avoid authentification issues, you should add `command: --default-authentication-plugin=mysql_native_password` under image (or somewhere else in the yml but at the same indentation level than image)

Reply:
In my tests this works fine with the new defaults. 

And as this is for development I don't see a problem with going with the new defaults.

Reply:
I think a few people have tested this and they didn't have problems with the new mysql8 auth method. 

I think we should embrace the new authentication in develop, if there are problems in the future at least we will know before it becomes a problem in production environments.